### ISSUE TITLE
data preprocessing

### ISSUE THREAD

**Issue by alvesisaque** [deeplearningunb/covid-cxr-test#1]
- [x] Collect data
- [x] Organize training set and test set 
- [x] cleaning the data
- [x] data split

**Comment by alvesisaque** [comment]
@egewarth  Faltou apresentar mais informações sobre os dados:

- [ ] Melhorar o gráfico que já temos (os gráficos que mostram a quantidade de imagens em cada)
- [x] Apresentar qual a diferença das imagens
